demos/storybook/storybook/stories/info-list-item/info-list-item.tsx

```diff
@@ -76,6 +76,31 @@ storiesOf('InfoListItem', module)
         ),
         notes
     )
+    .add(
```

Comment:
I might just incorporate this knob as part of the 'basic usage' story rather than dedicating a whole new story to third line

Reply:
Approving this PR...there will be a followup to create a full config story and move this there